libs/barista-components/radial-chart/src/radial-chart.ts

```diff
@@ -47,11 +51,15 @@ import {
   generatePieArcData,
   getSum,
 } from './utils/radial-chart-utils';
-import { DomSanitizer, SafeStyle } from '@angular/platform-browser';
 
 /** Size of the inner (empty) circle in proportion to the circle's radius. */
 const DONUT_INNER_CIRCLE_FRACTION = 0.8;
 
+/** Minimum width of the chart */
+const MIN_WIDTH = 400;
+/** Maximum width of the chart */
+const MAX_WIDTH = 800;
```

Comment:
Why do we now clamp the size of the svg now? This was not part of the issue discription or the API proposal. 
The container should take care of the sizing. Please remove this again.